tests/testsuite.src/run_misc.at

```diff
@@ -4355,6 +4355,32 @@ AT_CHECK([COB_LOAD_CASE=UPPER ./caller], [0], [], [])
 
 AT_CLEANUP
 
+AT_SETUP([CALL/CANCEL with COB_LOAD_GLOBAL])
+AT_KEYWORDS([runmisc CALL CANCEL COB_LOAD_GLOBAL])
+
+AT_DATA([callee.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      callee.
+       PROCEDURE        DIVISION.
+           EXIT PROGRAM.
+])
+
+AT_DATA([caller.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      caller.
+       PROCEDURE        DIVISION.
+           CALL "callee"
+           END-CALL.
+           CANCEL "callee".
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE_MODULE callee.cob], [0], [], [])
+AT_CHECK([$COMPILE caller.cob], [0], [], [])
+AT_CHECK([COB_LOAD_GLOBAL=true ./caller], [0], [], [])
+AT_CHECK([COB_LOAD_GLOBAL=false ./caller], [0], [], [])
+
```

Comment:
Can you please add both COB_LOAD_GLOBAL options (and its name to the keyword) for the pre-load test to have that covered as well?